workshop/Lab5/README.md

```diff
@@ -0,0 +1,227 @@
+# Lab 5 Deploy an OSB service and attach it to a deployment
+
+Learn to provision a service instance, create a binding for your service instance,
+and use that binding to connect your deployment to that service.
+
+# Before we begin
+
+If you skipped Lab0 or otherwise didn't create a Persistant Volume, we need one for Minibroker
+to bind the Redis instance to. Find the pv.yaml file in the [Lab0 directory](../Lab0], and run
+`kubectl create -f pv.yaml`. Note that while we need to do this because our broker is
+provisionin resources inside our Kubernetes cluster, this is not part of the normal
+OSB service workflow.
+
+# Provision a Service Instance
+
+Now that we have Minibroker up and running in our Kube cluster, we can use it to provision
+a redis instance. If we run `svcat get classes` we can see all the services offered by Minibroker:
+```console
+$ svcat get classes
+     NAME             DESCRIPTION             UUID     
++------------+---------------------------+------------+
+  mariadb      Helm Chart for mariadb      mariadb     
+  mongodb      Helm Chart for mongodb      mongodb     
+  mysql        Helm Chart for mysql        mysql       
+  postgresql   Helm Chart for postgresql   postgresql  
+  redis        Helm Chart for redis        redis    
+```
+
+The same can be done for the Plans of those Classes by running `svcat get plans`. Note that
+this response contains a bit more info, so it can display quite a lot of text.
+```console
+$ svcat get plans
+   NAME       CLASS               DESCRIPTION                   UUID        
```

Comment:
I don't see the uuid column - perhaps it was removed?

Reply:
Plus they're not uuid :-)

Reply:
yeah, that's been removed